scenes/game/local_game.gd

```diff
@@ -4281,8 +4384,9 @@ func handle_strike_effect(card_id :int, effect, performing_player : Player):
 			events += [create_event(Enums.EventType.EventType_Strike_EffectChoice, performing_player.my_id, 0, "EffectOption")]
 		"move_to_space":
 			var space = effect['amount']
+			var remove_buddies_encountered = effect['remove_buddies_encountered']
```

Comment:
does this need a default value? or is it only reachable through move_to_any_space